src/main/java/net/minecraftforge/common/capabilities/Capability.java

```diff
@@ -65,10 +65,29 @@
      * the fully qualified class name for the target interface.
      */
     public String getName() { return name; }
+
     /**
      * @return An instance of the default storage handler. You can safely use this store your default implementation in NBT.
      */
     public IStorage<T> getStorage() { return storage; }
+    
+    /**
+     * Quick access to the IStorage's readNBT. 
+     * See {@link IStorage#readNBT(Capability, Object, EnumFacing, NBTBase)}  for documentation.
+     */
+    public void readNBT(T instance, EnumFacing side, NBTBase nbt)
```

Comment:
It's consistent with every single NBT method we have everywhere else, though. If we want to change that practice, I'm all up for it, although I wouldn't call it just "read". If I were to remove NBT, I'd rename it to deserialize instead.